cue/data/gen.cue

```diff
@@ -191,6 +191,83 @@ Family: scuemata.#Family & {
                     // TODO docs
                     maxDataPoints?: number
 
+                    // TODO docs
+                    dashLength?: number | *10
```

Comment:
these are for graph panel, i know you said ignore them for now, i would revert this later on. Just fixed for the purpose of clearer errors when running the script.